security/src/main/java/edu/unc/lib/dl/acl/util/AccessGroupSet.java

```diff
@@ -62,7 +66,7 @@ public void addAccessGroup(String group) {
      * Determines is any of the objects contained within the specified
      * collection are present in the access group set. If the objects contain
      * pid prefixes, they are stripped off before checking
-     * 
+     *
      * @param c
      *            collection to be checked for matches.
```

Comment:
this comment should be moved up one line beside the  @param

Reply:
eclipse defaulted to putting descriptions of parameters on the following lines for some weird reason, so that stuff is scattered around.  trying to move away from it though